configs/Phi-3-mini-4k-instruct/README.md

```diff
@@ -46,4 +46,47 @@ python console_chat.py
 python gradio_chat.py
 ```
 
-To use `prompt flow` in VS Code, please refer to this [Quick Start](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html).
\ No newline at end of file
+To use `prompt flow` in VS Code, please refer to this [Quick Start](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html).
+
+
+## Remote Development
+### Prerequisites
+1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+2. If you are using private dataset on HuggingFace, make sure you have a [HuggingFace account](https://huggingface.co/) and [generate an access token](https://huggingface.co/docs/hub/security-tokens)
+3. If you enable [Weights & Biases](https://wandb.ai/site), make sure you have a Weights & Biases account and [retrieve your API Key](https://wandb.ai/authorize)
+
+### Provision Azure Resources
+To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette.
+
+You can monitor the progress of the provision via the link that is displayed in the output channel.
+
+### Add ACA Secrets
+#### Weights & Biases API Key
+If you've enabled Weights & Biases in your project, it's crucial to set your Weights & Biases API Key for seamless access. Use the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning` command to input your Weights & Biases API Key. In this process, the secret name should be [`WANDB_API_KEY`](https://docs.wandb.ai/guides/track/environment-variables#optional-environment-variables) with your API Key as the secret value.
+
+#### [Optional] HuggingFace token
+If you're using private HuggingFace dataset, you should set your HuggingFace token as an environment variable to bypass the need for manual login on the Hugging Face Hub.
```

Comment:
Proposed:
If you're using private HuggingFace dataset, set your HuggingFace token as an environment variable to avoid the need for manual login on the Hugging Face Hub.